jaxopt/_src/armijo_sgd.py

```diff
@@ -224,7 +224,7 @@ class ArmijoSGD(base.StochasticSolver):
   maxiter: int = 500
   maxls: int = 15
   tol: float = 1e-3
-  verbose: int = 0
+  verbose: bool = False
```

Comment:
The idea of using an `int` was that we could have several levels of verbosity:
- 1: verbose
- 2: very verbose
- 3: very very verbose

For example, 2 could be used to print linesearch information.

Do you think it's unnecessary?

Reply:
I did a two level verbose for the solvers with linesearch as you suggested. 
I made all verbose types to be boolean or int to be consistent.
Thanks for the suggestion!